diff --git a/node_modules/react-native-reanimated/lib/module/ReanimatedModule/NativeReanimated.js b/node_modules/react-native-reanimated/lib/module/ReanimatedModule/NativeReanimated.js
index 7ee916a..1c088fd 100644
--- a/node_modules/react-native-reanimated/lib/module/ReanimatedModule/NativeReanimated.js
+++ b/node_modules/react-native-reanimated/lib/module/ReanimatedModule/NativeReanimated.js
@@ -47,7 +47,7 @@ class NativeReanimatedModule {
       throw new ReanimatedError(`Native part of Reanimated doesn't seem to be initialized.
 See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#native-part-of-reanimated-doesnt-seem-to-be-initialized for more details.`);
     }
-    if (!globalThis.RN$Bridgeless && !SHOULD_BE_USE_WEB) {
+    if (__DEV__ && !globalThis.RN$Bridgeless && !SHOULD_BE_USE_WEB) {
       throw new ReanimatedError('Reanimated 4 supports only the React Native New Architecture and web.');
     }
     if (__DEV__) {
diff --git a/node_modules/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts b/node_modules/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts
index 0d3fe2e..c8b6f92 100644
--- a/node_modules/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts
+++ b/node_modules/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts
@@ -83,7 +83,7 @@ class NativeReanimatedModule implements IReanimatedModule {
 See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#native-part-of-reanimated-doesnt-seem-to-be-initialized for more details.`
       );
     }
-    if (!globalThis.RN$Bridgeless && !SHOULD_BE_USE_WEB) {
+    if (__DEV__ && !globalThis.RN$Bridgeless && !SHOULD_BE_USE_WEB) {
       throw new ReanimatedError(
         'Reanimated 4 supports only the React Native New Architecture and web.'
       );